src/web_server/serve_static.cc

```diff
@@ -21,9 +21,12 @@ inline std::string_view mime_type(beast::string_view path) {
     }
     return path.substr(pos);
   }();
-  if (iequals(ext, ".js")) {
+  if (iequals(ext, ".js") || iequals(ext, ".mjs")) {
     return "application/javascript";
```

Comment:
According to MDN, the preferred mime type for JavaScript is `text/javascript`: https://developer.mozilla.org/en-US/docs/Web/HTTP/Guides/MIME_types/Common_types. In my testing, `application/javascript` also works well.